sdktests/testapi_hooks.go

```diff
@@ -78,3 +84,28 @@ func (h *Hooks) ExpectCall(t *ldtest.T, hookName string,
 		}
 	}
 }
+
+// ExpectAtLeastOneCallForEachHook waits for a single call from N hooks. If there are fewer calls recorded,
+// the test will fail. However, this helper cannot detect if there were more calls waiting to be recorded.
+func (h *Hooks) ExpectAtLeastOneCallForEachHook(t *ldtest.T, hookNames []string) []servicedef.HookExecutionPayload {
+	out := make(chan o.Maybe[servicedef.HookExecutionPayload])
+
+	totalCalls := len(hookNames)
+
+	for _, hookName := range hookNames {
+		go func(name string) {
+			out <- helpers.TryReceive(h.instances[name].hookService.CallChannel, hookReceiveTimeout)
+		}(hookName)
+	}
+
+	payloads := make([]servicedef.HookExecutionPayload, totalCalls)
```

Comment:
I think this is incorrect. It makes a slice of the length of expected calls. Then it appends to that list which will then always be longer than the expected calls.

Reply:
```suggestion
	payloads := make([]servicedef.HookExecutionPayload, 0)
```

Reply:
Yea, should have been `make(t, 0, totalCalls)`.